app/models/results/csv/generator.rb

```diff
@@ -0,0 +1,168 @@
+module Results
```

Comment:
This file shows as totally new because I split the classes out into separate files. See the individual commits if you want to see what got changed. Mostly caching and eager loading stuff.